core/src/main/java/org/elasticsearch/search/aggregations/bucket/adjacency/AdjacencyMatrixAggregationBuilder.java

```diff
@@ -57,15 +57,15 @@
     private String separator = DEFAULT_SEPARATOR;
 
     public static Aggregator.Parser getParser() {
```

Comment:
Can this be
```
public static final Aggregator.Parser PARSER;
static {
  ObjectParser objectParser = new ObjectParser();
  // build the parser
  PARSER = (name, parser) -> {
    // Stuff
  };
}
```

I think that'd be closer to how we parse other things. Not the same, but closer.